core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -6069,19 +6071,19 @@ class KafkaApisTest {
   @Test
   def testRaftShouldAlwaysForwardCreateTokenRequest(): Unit = {
     metadataCache = MetadataCache.kRaftMetadataCache(brokerId)
-    verifyShouldAlwaysForwardErrorMessage(createKafkaApis(raftSupport = true).handleCreateTokenRequest)
+    verifyShouldAlwaysForwardErrorMessage(createKafkaApis(raftSupport = true).handleCreateTokenRequestZk)
```

Comment:
I think this change is incorrect.  We are checking that a KRaft broker forwards to the KRaft controller, and {handleCreateTokenRequestZk, handleRenewTokenRequestZk,handleExpireTokenRequestZk} aren't invoked on KRaft brokers.

Reply:
It is testing the error message in the case where the request was not forwarded and we are in KRaft mode.

Reply:
I added a comment